pom.xml

```diff
@@ -22,8 +25,7 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <version.jmh>1.26</version.jmh>
-    <!-- This is here so we can override the version from parent, i.e. benchmark other version -->
-    <kie.bom.version>${version.org.kie}</kie.bom.version>
+    <kie.bom.version>7.69.0-SNAPSHOT</kie.bom.version>
```

Comment:
AFAIK there is no grouping KIE BOM on 8.x, so I think we should remove this property together with the dependency, right?

Reply:
Yes, I kept it because one of the benchmarks depends on a class which cannot be found in 8.x, but I have to investigate it and maybe refactoring this specific benchmark.